smart_campus/app/views.py

```diff
@@ -488,3 +589,41 @@ def get_all_travel_plans(request):
     ]
 
     return JsonResponse(data={'data': data}, status=200, json_dumps_params={'ensure_ascii': False}, content_type='application/json; charset=utf-8')
+
+
+@login_required
+def reward_list_page(request):
+    # list all rewards
+
+    context = {
+        'email': request.user.email,
+        'rewards': Reward.objects.all(),
+        'categories': StationCategory.objects.all()
+    }
+
+    return render(request, 'app/reward_list_page.html', context)
+
+
+@login_required
+def reward_add_page(request):
+    # add the reward
+    if request.method == 'POST':
+        reward_form = PartialRewardForm(request.POST, request.FILES)
+
+        if reward_form.is_valid():
+            reward_form.save()
+
+            context = {
+                'rewards': Reward.objects.all(),
+                'email': request.user.email,
+                'categories': StationCategory.objects.all()
+            }
+            # messages.info(request, 'Three credits remain in your account.')
+            # return HttpResponse('Success', status=200)
+            return render(request, 'app/reward_list_page.html', context)
+
+    context = {
+        'email': request.user.email,
+        'categories': StationCategory.objects.all()
+    }
+    return render(request, 'app/reward_add_page.html', context)
```

Comment:
Unify your blank lines style.
Some of your view functions have the blank line between `context` and `return ...`.

Reply:
okay, fixed it.